ui-components/src/overlays/Modal.tsx

```diff
@@ -59,7 +59,6 @@ export const Modal = (props: ModalProps) => {
       onClose={props.onClose}
       className={props.className}
       backdrop={props.backdrop}
-      scrollable={props.scrollable}
```

Comment:
I think we might still need to pass this to Overlay

Reply:
Do you mind expanding on this? As far as I could tell, the single scrollable prop for the modal would always result in the page scrolling after the modal scrolls which didn't seem intended. The video shows it passed to overlay.

https://user-images.githubusercontent.com/53269332/188717800-349708ef-5d7f-4f90-8c7f-64e4226ffb00.mov

.

Reply:
@ColinBuyck Good call let me connect w SF and make sure they didn't want the background scroll for some reason!

Reply:
There is a case in SF where they want the background to be able to scroll (see [here](https://dahlia-full.herokuapp.com/listings/a0W4U00000KnHO6UAN) and click on the `view lottery results` button in the right panel) - I wonder if the difference here is that the modal in SF is not itself scrollable. I _think_ if we stop passing the scroll to Overlay this won't behave that way anymore. We may need to split the scroll behavior into two props (`scrollableModal` and `scrollableOverlay` or something similar).

Reply:
I agree on all of this! I am good to split the scroll behavior for Detroit and then we can establish how to communicate the breaking change as part of the Core sync PR. Does that make sense to you @emilyjablonski?